hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/flow/txn/modules/LastHandledTime.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.workflows.handle.flow.txn.modules;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import javax.inject.Qualifier;
+
+/**
+ * Qualifier to distinguish between two Instant values provided by Dagger.
+ * We annotate this qualifier for the last handled time.
+ */
+@Target({METHOD, PARAMETER, TYPE})
+@Retention(RUNTIME)
+@Documented
+@Qualifier
+public @interface LastHandledTime {}
```

Comment:
I get that this annotation is for distinguishing Instants, but shortening it to `LastHandledTime` – I suspect that people will ask what "last handled time" is a lot in the future

Reply:
This is just provided by `BlockRecordManagerImpl.lastHandledTime`. I will add a comment. Thanks @mhess-swl